### ISSUE TITLE
Authentication failed. You have not been accepted into the beta.

### ISSUE THREAD

**Issue by Liu-Angelo** [acheong08/EdgeGPT#13]
— OS：linux
— python version：Python 3.10.4

python3 -m EdgeGPT -h

        EdgeGPT - A demo of reverse engineering the Bing GPT chatbot
        Repo: github.com/acheong08/EdgeGPT
        By: Antonio Cheong

        !help for help

        Type !exit to exit
        Enter twice to send message
    
Initializing...
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/requests/models.py", line 971, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/local/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/EdgeGPT.py", line 130, in __init__
    self.struct = response.json()
  File "/usr/local/lib/python3.10/site-packages/requests/models.py", line 975, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.10/site-packages/EdgeGPT.py", line 309, in <module>
    asyncio.run(main())
  File "/usr/local/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/local/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.10/site-packages/EdgeGPT.py", line 270, in main
    await bot.start()
  File "/usr/local/lib/python3.10/site-packages/EdgeGPT.py", line 216, in start
    self.conversation = Conversation()
  File "/usr/local/lib/python3.10/site-packages/EdgeGPT.py", line 132, in __init__
    raise Exception("Authentication failed. You have not been accepted into the beta.") from json.decoder.JSONDecodeError
TypeError: JSONDecodeError.__init__() missing 3 required positional arguments: 'msg', 'doc', and 'pos'

**Comment by acheong08** [comment]
Duplicate
- https://github.com/acheong08/EdgeGPT/issues/4

TL;DR: `pip3 uninstall EdgeGPT && pip3 install EdgeGPT --upgrade`
